scaleway/resource_k8s_cluster.go

```diff
@@ -385,8 +385,13 @@ func resourceScalewayK8SClusterCreate(ctx context.Context, d *schema.ResourceDat
 	}
 
 	d.SetId(newRegionalIDString(region, res.ID))
-
-	_, err = waitK8SClusterPool(ctx, k8sAPI, region, res.ID, d.Timeout(schema.TimeoutCreate))
+	if clusterType.(string) == "multicloud" {
```

Comment:
Can we set the types on the SDK? or as const values

Reply:
Nope they are like DEV1-S. We don't enum them because they can change more often.